wpengine/faustjs
# Feat: Output faust version in dev|build|start commands.

## Tasks


## Description

Outputs the Faust version number when running dev|build|start for debugging purposes.

## Related Issue(s):



## Testing

1. Build the cli package.
2. Run dev|build|start commands on the example projects. It should print the current projects faustwp-core and cli versions in the console.

## Screenshots


<img width="645" alt="Screenshot 2024-04-11 at 12 08 50" src="https://github.com/wpengine/faustjs/assets/328805/fa6db0ce-98b3-4b4a-823b-d449eb9f1100">

## Documentation Changes



## Dependant PRs
